### ISSUE TITLE
UnhandledPromiseRejectionWarning:   TypeError: execa.shellSync is not a function

### ISSUE THREAD

**Issue by mesqueeb** [quasarframework/app-extension-icon-genie#95]
I got this error:
```
app:build Extension(@quasar/icon-genie): Running beforeBuild hook... +3ms


Splashscreens for Cordova requires a Cordova plugin
(cordova-plugin-splashscreen) which was not found.

Attempting to install it...

(node:30988) UnhandledPromiseRejectionWarning:   TypeError: execa.shellSync is not a function

  - index.js:65 updateCordovaConfig
    [cinematch]/[@quasar]/quasar-app-extension-icon-genie/src/index.js:65:11
```

**Comment by rstoenescu** [comment]
Huh?

What Nodejs version are you on?

**Comment by mesqueeb** [comment]
I'm on `v12.12.0`

A quick fix is to just go into `src-cordova` and manually install `cordova-plugin-splashscreen` which resolves this issue.

**Comment by rstoenescu** [comment]
Fixed in v1.1.2.
